src/hooks/useGetTokens.ts

```diff
@@ -79,6 +27,7 @@ export function useGetTokens(networkId: Network | undefined, tableState: TableSt
     )?.averagePrice
 
     return {
+      lastDayUsdTimestamp: lastDayTimestampFrom,
```

Comment:
what's this `lastDayUsdTimestamp` value? is that the Last Updated timestamp value? If so, shouldn't this be picked from the `timestamp` value response?